webapp/app.py

```diff
@@ -401,42 +401,41 @@ def create_app():
         return res
 
     @web.middleware
-    async def error_formater(request, handler, **kwargs):
-        #pylint: disable=broad-except
+    async def error_formatter(request, handler, **kwargs):
+        # pylint: disable=broad-except
         def build_error(detail, status):
             errors = {"detail": detail, "status": status}
             return {"errors": [errors]}
 
+        original_error = ''
```

Comment:
what is the purpose of this variable? it's a string type and few lines below this variable changes to dict

Reply:
it was added by mistake :smile: